virtual_rainforest/core/config.py

```diff
@@ -17,6 +17,7 @@
 import tomli_w
 from jsonschema import FormatChecker
 
+import virtual_rainforest.core.variables as variables
 from virtual_rainforest.core.exceptions import ConfigurationError
```

Comment:
It is not identical. The method I used does not import the contents of the module until `setup_variables` is used deep in the code. This avoids circular import errors since by the time it is necessary to really load the contents of `variable` all the dependencies are already loaded. 

And it keep the namespace clean 😃